vector/src/main/java/im/vector/app/features/spaces/SpaceSummaryController.kt

```diff
@@ -132,13 +134,13 @@ class SpaceSummaryController @Inject constructor(
         }
 
         rootSpaces
-                ?.sortedBy { it.displayName }
+                ?.sortedBy { it.roomId }
```

Comment:
sorting by `roomId` is strange...

Reply:
yeah, for now it's just to be consistent across client, pending MSC